mohawk/tests.py

```diff
@@ -609,3 +613,57 @@ def test(self):
         sender.accept_response(receiver.response_header,
                                content=content,
                                content_type=content_type)
+
+
+class TestBewit(Base):
+
+    # Test cases copied from
+    # https://github.com/hueniverse/hawk/blob/492632da51ecedd5f59ce96f081860ad24ce6532/test/uri.js
+
+    def setUp(self):
+        self.credentials = {
+            'id': '123456',
+            'key': '2983d45yun89q',
+            'algorithm': 'sha256',
+        }
+
+    def test_bewit(self):
+        res = Resource(url='https://example.com/somewhere/over/the/rainbow',
+                       method='GET', credentials=self.credentials,
+                       timestamp=1356420407 + 300,
+                       nonce='',
+                       )
+        bewit = get_bewit(res)
+
+        mac = calculate_mac('bewit', res, None).decode('ascii')
+        eq_(b64decode(bewit).decode('ascii'), '123456\\1356420707\\{mac}\\'.format(mac=mac))
+        eq_(bewit, "MTIzNDU2XDEzNTY0MjA3MDdcSUdZbUxnSXFMckNlOEN4dktQczRKbFdJQStValdKSm91d2dBUmlWaENBZz1c")
```

Comment:
I'm a little iffy on that but I see your point. If that's the case, I'd suggest hard coding the `mac` into the line above it rather than calculating a mac. With that, no base64 comparison is needed. The reason I don't like the base64 comparison is that when you see it in a test failure the first thing you'd need to do is decode it.